src/executor/graph_executor.cc

```diff
@@ -1204,14 +1203,17 @@ void GraphExecutor::InitCachedOps() {
         LOG(INFO) << "node " << nid << " var";
       } else {
         LOG(INFO) << "node " << nid << " " << inode.source->attrs.op->name;
+        const auto& vstorage_type = graph_.GetAttr<StorageTypeVector>("storage_type");
```

Comment:
It is only required in a smaller scope, so no need fetching it if it isn't going to be used.